### PR TITLE
docs: ✏️ Sidekiq instrumentation options

### PR DESCRIPTION
**Pull request by kohbis**
For users of Sidekiq instrumentation, I thought it would be nice to have a list of options and their uses.
So I added a (simple explain and format) table in the README ✏️ 
（At least no code reading required）

I improve any further explanations, items, or errors that are necessary.

If you don't need it, you can close it 🙏

### CODE REVIEW

File: instrumentation/sidekiq/README.md [hunk 1]
```diff
@@ -29,6 +29,27 @@ OpenTelemetry::SDK.configure do |c|
...
+
+If you wish to change any of the default behaviour, you can pass options during instrumentation setup:
+
+```ruby
+OpenTelemetry::SDK.configure do |c|
+  c.use 'OpenTelemetry::Instrumentation::Sidekiq', span_naming: :job_class
+end
+```
+
+|name|type|default|description|
```
Comment (arielvalentin) [inline instrumentation/sidekiq/README.md:43]: ✨ Thank you!

**Review by arielvalentin — APPROVED** [review]
Thank you for your contribution! 

If you are curious and would be interested in helping us a little more in this area, I would be interested in seeing if we could somehow generate this markdown README as well as some yardoc that may appear here:

https://www.rubydoc.info/gems/opentelemetry-instrumentation-sidekiq/OpenTelemetry/Instrumentation/Sidekiq/Instrumentation

That way we could ensure our docs are up-to-date with the options we provide. 

Is that something that would interest you?

**Comment by kohbis** [comment]
@arielvalentin Thank you for confirming!

> If you are curious and would be interested in helping us a little more in this area, I would be interested in seeing if we could somehow generate this markdown README as well as some yardoc that may appear here:
Sorry, I missed the mention of Yard in the Contribute guide.

I don't know how to output to README, I will first rewrite it from README to Yard so that we can check it in rubydoc 🙏

**Comment by arielvalentin** [comment]
Thank you for taking the time to do that!

**Comment by arielvalentin** [comment]
@kohbis Would you be interested and available to help us do the same for other instrumentations?

**Comment by kohbis** [comment]
> @kohbis Would you be interested and available to help us do the same for other instrumentations?

@arielvalentin If you don't mind if I proceed a little at a time, I'd be happy to help 🙏

**Pull request merged**